roles/kube_prometheus_stack/vars/main.yml

```diff
@@ -75,8 +75,7 @@ _kube_prometheus_stack_helm_values:
       enabled: true
       servicePort: 8081
       ingressClassName: "{{ kube_prometheus_stack_ingress_class_name }}"
-      annotations:
-        cert-manager.io/cluster-issuer: atmosphere
+      annotations: "{{ kube_prometheus_stack_ingress_annotations | combine(ingress_global_annotations, recursive=True) }}"
```

Comment:
This variable will not exist, I think you need to maybe move some of these into `vars` and then also make sure there is some form of defaults since it may not be defined.

Reply:
@mnaser ok, i see
btw, i think we can rename `ingress_global_annotations` as  `ingress_default annotations` which is a role-level variable.
And set its default value with a new global variable `atmosphere_ingress_annotations`.\

what do you think about this?

Reply:
If we do that, we have users that already depend on it, so we need to make sure that we allow users to use the old variable.